src/components/media-player/ha-media-player-browse.ts

```diff
@@ -726,6 +726,10 @@ export class HaMediaPlayerBrowse extends LitElement {
           background-color: transparent;
         }
 
+        mwc-list-item .title {
+          margin-left: 10px;
+        }
```

Comment:
Why is this needed? `mwc-list-item` should handle styling itself.

Reply:
Without it, the song title is directly next to the song icon, which doesn't look good. With that change there is a bit of space (see screenshot).

Reply:
I don't see that here?
![image](https://user-images.githubusercontent.com/5662298/92459247-b9213000-f1c6-11ea-8383-286f8c5113b0.png)

Reply:
This is how it looks for me (Spotify Recently Played) in the "dev" branch:
![image](https://user-images.githubusercontent.com/114137/92461165-377ed180-f1c9-11ea-8e59-d652062f59f6.png)

Reply:
![image](https://user-images.githubusercontent.com/15093472/92461898-15d21a00-f1ca-11ea-97fe-1684f15a363c.png)
Mine looks correct (Spotify Recently Played)

Reply:
It seems to be browser dependent. In Chrome the current dev coding looks good, but not in FF 80.0.1. When I apply my changes, then in Chrome the text moves even further away from the icon, so some margins seem to stack.

Reply:
A quick glance shows, that this is already adding space between icon and title in Chrome. In FF this doesn't seem to be there:
![image](https://user-images.githubusercontent.com/114137/92463615-56329780-f1cc-11ea-903f-756180440277.png)

Reply:
What happens when you set that variable to 16px?

Reply:
The whole red marked CSS block is not present in FF inspector, so it is not a matter of `--mdc-list-item-graphic-margin` being undefined, but it missing completely. The one below the red marked one, does exist in FF. The variables that are used there (e.g. `--mdc-list-item-graphic-size`) are not set in FF, but apparently in Chrome.

Reply:
Weird that this hasn't been brought up in the upstream repos...

Reply:
OK, then set `--mdc-list-item-graphic-margin` to 0, and also check for RTL support.

Reply:
Were any of you able to confirm my issue in FF on your instances?

Reply:
Can confirm it's definitely an issue in Firefox.

<img width="350" alt="Screen Shot 2020-09-08 at 12 01 34 PM" src="https://user-images.githubusercontent.com/9065557/92500547-49617400-f1cb-11ea-87cf-03cb1bba3c24.png">

Reply:
@bramkragten Added the fixes and verified RTL support. Let me know if I did it in the wrong way :smile: .